src/stories/product-switch/product-switch.stories.ts

```diff
@@ -0,0 +1,119 @@
+import { moduleMetadata } from '@storybook/angular';
+import { withKnobs, boolean, select, text, object } from '@storybook/addon-knobs';
+import { withA11y } from '@storybook/addon-a11y';
+import { ProductSwitchComponent, ProductSwitchModule } from 'libs/core/src/lib/product-switch/public_api';
+
+export default {
+    title: 'Fd product-switch',
+    component: ProductSwitchComponent,
+    moduleMetadata: moduleMetadata,
+    decorators: [
+        withKnobs,
+        withA11y,
+        moduleMetadata({
+            imports: [ProductSwitchModule],
+            declarations: []
+        })
+    ]
+};
+
+
+
+export const ProductSwitch = () => ({
+    template:
+        `<fd-product-switch 
+                [closeOnEscapeKey]="closeOnEscapeKey"
+                [closeOnOutsideClick]="closeOnOutsideClick"
+                [disabled]="disabled"
+                [fillControlMode]="fillControlMode"
+                [focusTrapped]="focusTrapped"
+                [isDropdown]="isDropdown"
+                [isOpen]="isOpen"
+                [noArrow]="noArrow"
+                [placement]="placement"
+                [triggers]="triggersVar">
+            <fd-product-switch-body
+                [forceListMode]="forceListModeVar"
+                [products]="objectVar"
+                [dragAndDropEnabled]="dragAndDropEnabled">
+            </fd-product-switch-body>
+        </fd-product-switch>
+    `,
+    props: {
+        forceListModeVar: boolean('Mobile Mode', false),
+        closeOnEscapeKey: boolean('Close On Esc', false),
```

Comment:
The correct name would be `onCloseKeyPressed`
in general, when you deal with events `on` prefix should be added